src/instructlab/eval/mt_bench_common.py

```diff
@@ -87,32 +87,15 @@ def load_questions(question_file: str, begin: Optional[int], end: Optional[int])
     return questions
 
 
-def load_model_answers(answer_dir: str, model_name=None, answer_file=None) -> dict:
-    """Load model answers.
+def load_model_answers(answer_file, model_name=None) -> dict:
```

Comment:
We've lost the type hint here - could this be multiple different types? It seems from the `_load_answers` func definition this always needs to be a JSON file: https://github.com/danmcp/eval/blob/62e4bc3e5dee05958cab2182f40cd8b6df15cbec/src/instructlab/eval/mt_bench_common.py#L102

Reply:
answer_file didn't have a type hint before (answer_dir did), but happy to add one.

Reply:
Ah you're right, thanks for the addition!